doc/man/cockpit.conf.xml

```diff
@@ -149,6 +149,14 @@ ForwardedForHeader = X-Forwarded-For
           <code>/cockpit/</code> and <code>/cockpit+new/</code> are not.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><option>WebsocketRoot</option></term>
```

Comment:
I would put here `EXPERIMENTAL: Might change or be dropped in future releases`

Reply:
Then maybe let's just not document it at all yet?